src/pycram/costmaps.py

```diff
@@ -753,7 +753,7 @@ class SemanticCostmap(Costmap):
     table surface.
     """
 
-    def __init__(self, object, urdf_link_name, size=100, resolution=0.02, world=None):
+    def __init__(self, object, urdf_link_name, size=100, resolution=0.02, world=None, margin_cm=0.2):
```

Comment:
Something like margin_cm should just be margin: Unit. This way people can write whatever unit they want there. Consider using some metric package here like https://github.com/hgrecco/pint